tests/datasets/test_wms.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+import requests
+
+from torchgeo.datasets import WMSDataset
+
+SERVICE_URL = "https://mesonet.agron.iastate.edu/cgi-bin/wms/nexrad/n0r-t.cgi?"
+
+
+def service_ok(url: str, timeout: int = 5) -> bool:
+    try:
+        resp = requests.head(url, allow_redirects=True, timeout=timeout)
+        ok = bool(resp.ok)
```

Comment:
The `ok` property of `Response` always returns a bool so casting is not necessary.